program.c

```diff
@@ -156,11 +155,11 @@ int program_execute(struct qdl_device *qdl, int (*apply)(struct qdl_device *qdl,
 int program_find_bootable_partition(void)
 {
 	struct program *program;
-	const char *label;
+	char *label;
```

Comment:
why?

Reply:
Why we should use const, when pointer holds dynamically allocated data and dynamically assigned value?

Reply:
I have updated patch with commit message with all reasons.